.github/ISSUE_TEMPLATE/bug_report_devtools_extension.md

```diff
@@ -0,0 +1,34 @@
+---
+name: Bug report DevTools extension
+about: Create a report to help improve drift's DevTools extension
+title: ''
+labels: ['package-devtools']
+assignees: ''
+
+---
+
+**Describe the bug**
+
+A clear and concise description of what the bug is, what behavior you saw. If this is
+a feature request, ignore delete this.
+
+**Expected behavior**
+
+Please briefly describe what behavior you expected
+
+**Reproducing**
+
+How the bug can be reproduced. If this is a feature request, ignore delete this.
```

Comment:
If we want to use this for both feature requests and bug reports, I think the sections should also be a little more generic?

E.g. what if the body was just a reminder of what to do instead of a prestructured issue? E.g.

```
Thanks for taking the time to file an issue for drift's DevTools extension!

For feature requests, please describe the feature you're interested in and perhaps how you would expect it to be integrated into the extension.
For bug reports, please include the relevant `drift` and `flutter` versions. A screenshot may also be helpful.
```

Reply:
Please review it. I think we can leave the screenshots version and the other topic below because sometimes people using the DevTools from the `drift` package itself might be some versions behind and asking for features that already exist in newer versions so that would be helpful anyway. And about the screenshots and discussion, if that doesn't apply the person can simply remove it or fill it with "N/A" or something.